modules/core/04-channel/v2/keeper/msg_server_test.go

```diff
@@ -360,7 +361,7 @@ func (suite *KeeperTestSuite) TestMsgAcknowledgement() {
 		{
 			name: "success: NoOp",
 			malleate: func() {
-				suite.chainA.App.GetIBCKeeper().ChannelKeeperV2.SetPacketCommitment(suite.chainA.GetContext(), recvPacket.SourceChannel, recvPacket.Sequence, []byte{})
+				suite.chainA.App.GetIBCKeeper().ChannelKeeperV2.DeletePacketCommitment(suite.chainA.GetContext(), recvPacket.SourceChannel, recvPacket.Sequence)
```

Comment:
This is unrelated but I caught it while editing the test below. While the previous also works, this is cleaner